components/expenses/ExpenseForm.js

```diff
@@ -219,7 +233,9 @@ const ExpenseFormBody = ({
   const stepOneCompleted =
     values.payoutMethod &&
     isEmpty(flattenObjectDeep(omit(errors, 'payoutMethod.data.currency'))) &&
-    (isReceipt ? true : values.payeeLocation?.country && values.payeeLocation?.address);
+    (isReceipt || values.payee.type === CollectiveType.COLLECTIVE
+      ? true
+      : values.payeeLocation?.country && values.payeeLocation?.address);
```

Comment:
:ice_cream: This whole thing starts to get difficult to read. Would be great to introduce a dedicated function with if/else

Reply:
I've added a new function, `checkAddressValuesAreCompleted` to make it a bit more readable. Let me know if thinks looks better now. 😉